mobile/src/calendar/calendar-page.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { Component, Fragment } from 'react';
+import moment, { Moment } from 'moment';
+import { View, StyleSheet, TouchableHighlight, LayoutChangeEvent } from 'react-native';
+import { StyledText } from '../override/styled-text';
+import { calendarStyles, calendarPeriodStyles, calendarPeriodColors } from './styles';
+import { DayModel, WeekModel, PeriodsModel, PeriodModel } from '../reducers/calendar/calendar.model';
+import { StartPeriod, EndPeriod, Period, DotPeriod } from './calendar-page-period';
+import { CalendarEventsType } from '../reducers/calendar/calendar-events.model';
+import { WeekDay, WeekDayCircle } from './calendar-page-weekday';
+
+export type OnSelectedDayCallback = (day: DayModel) => void;
+
+interface CalendarPageDefaultProps {
+    hidePrevNextMonthDays?: boolean;
+    periods?: PeriodsModel;
+}
+
+interface CalendarPageProps {
+    weeks: WeekModel[];
+    onSelectedDay: OnSelectedDayCallback;
+}
+
+interface CalendarPageState {
+    weekHeight: number;
+}
+
+export class CalendarPage extends Component<CalendarPageDefaultProps & CalendarPageProps, CalendarPageState> {
+    public static defaultProps: CalendarPageDefaultProps = {
+        hidePrevNextMonthDays: false
+    };
+
+    private readonly weekdaysNames = moment()
+        .locale('en')
```

Comment:
Shouldn't we use device locale?

Reply:
We sort of decided to use only 'en'. All design sketchs use en locale.